src/ethereum_spec_tools/evm_tools/t8n/t8n_types.py

```diff
@@ -268,22 +268,23 @@ class Result:
     requests_hash: Optional[Hash32] = None
     requests: Optional[List[Bytes]] = None
 
-    def get_receipts_from_tries(
-        self, t8n: Any, tx_trie: Any, receipts_trie: Any
+    def get_receipts_from_output(
+        self,
+        t8n: Any,
+        block_output: Any,
     ) -> List[Any]:
         """
         Get receipts from the transaction and receipts tries.
         """
         receipts: List[Any] = []
-        for index in tx_trie._data:
-            if index not in receipts_trie._data:
-                # Meaning the transaction has somehow failed
-                return receipts
+        for key in block_output.receipt_keys:
```

Comment:
I'm getting an `AttributeError` on this line when I run the tests.

Reply:
Ah. Missed pushing the commit which has the back-ports to older forks. Should be good now